### ISSUE TITLE
Does not see the xorg-x11-drv-dummy

### ISSUE THREAD

**Issue by DJs3000** [Xpra-org/xpra#3823]
Hello. I'm trying to build xpra on my system. My system is similar to Debian and has few packages in its repository. I have to build a lot from sources. I build xpra like this:
```shell
git clone --recursive https://github.com/Xpra-org/xpra
cd xpra
python3 ./setup.py install --prefix=/usr
```
After some time I get an error:
```
rpm query: out=пакет xorg-x11-drv-dummy not installed
, err=
rpm query: out=пакет xf86-video-dummy not installed
, err=
found dummy driver version (0,)
copying fs/etc/xpra/xorg-uinput.conf -> /etc/xpra (0o644)
chmod(/etc/xpra/xorg-uinput.conf, 0o644)
copying fs/etc/xpra/xorg.conf -> /etc/xpra (0o644)
chmod(/etc/xpra/xorg.conf, 0o644)
copying fs/etc/X11/xorg.conf.d/90-xpra-virtual.conf -> /etc/X11/xorg.conf.d/ (0o644)
chmod(/etc/X11/xorg.conf.d/90-xpra-virtual.conf, 0o644)
copying fs/etc/pam.d/xpra -> /etc/pam.d (0o644)
chmod(/etc/pam.d/xpra, 0o644)
copying fs/etc/sysconfig/xpra -> /etc/sysconfig (0o644)
chmod(/etc/sysconfig/xpra, 0o644)
copying fs/etc/init.d/xpra -> /etc/init.d (0o644)
chmod(/etc/init.d/xpra, 0o644)
copying fs/etc/dbus-1/system.d/xpra.conf -> /etc/dbus-1/system.d (0o644)
chmod(/etc/dbus-1/system.d/xpra.conf, 0o644)
./docs               -> /usr/share/doc/xpra/
./docs/Build         -> /usr/share/doc/xpra/Build
  Dependencies.md      -> Dependencies.html
error: [Errno 2] No such file or directory: 'pandoc'
```
i build and install(/usr) - `https://gitlab.freedesktop.org/xorg/driver/xf86-video-dummy`
But xpra doesn't see the `xorg-x11-drv-dummy`

What can I try to do?

**Comment by totaam** [comment]
Assuming that xpra did not detect that you have `Xorg` installed and available (which is unlikely):
Build with:
```bash
python3 ./setup.py --with-Xdummy
```

This will ensure that your config uses `Xdummy` instead of `Xvfb`.

Ideally, we should provide a way to override the dummy version detection too.

**Comment by totaam** [comment]
The commit above should allow you to do:
```shell
./setup.py --dummy-driver-version=0.4 build
```
(your are responsible for specifying the correct version here - 0.4 is recommended, older versions can't do randr)

**Comment by DJs3000** [comment]
I rebuild xorg-x11-drv-dummy to version 0.4. Now log:
  
![image](https://user-images.githubusercontent.com/16526540/231797952-3813cc91-b0f6-4fbe-b508-0722645e48f2.png)

**Comment by totaam** [comment]
Already fixed in e72a44e09caac7c3ee9af72db814a7b4cfdbf682.
(please don't use pictures for text content)
